nntrainer/compiler/tflite_opnode.cpp

```diff
@@ -61,12 +112,78 @@ void TfOpNode::setLayerNode(const LayerNode &layer) {
 
 void TfOpNode::setWeightTransformFn(TransformFn fn) { weight_transform = fn; }
 
+void TfOpNode::setInputTransformFn(TransformFn fn) { input_transform = fn; }
+
+void TfOpNode::weightReorder(unsigned int node_count) {
+
+  if (need_reorder_weight == true) {
+
+    auto previous_input_shape = input_nodes[0]->getInputs()[0];
+
+    const unsigned int UNIT = outputs[0]->height();
+    const unsigned int CHANNEL = previous_input_shape->channel();
+    const unsigned int HEIGHT = previous_input_shape->height();
+    const unsigned int WIDTH = previous_input_shape->width();
+
+    auto weight_data = weights[0]->getData();
+    auto *ptr = const_cast<float *>(weight_data);
```

Comment:
```suggestion
    flaot *ptr = weights[0]->getData();
```
Isn't it possible? float is default for getData.

Reply:
There is non-const version of getData. please check.
https://github.com/nnstreamer/nntrainer/blob/main/nntrainer/tensor/tensor.h#L1197